app/views/listings/show.html.erb

```diff
@@ -1,4 +1,122 @@
+<div id="wrapper">
 
+        <!-- Sidebar -->
+        <div id="sidebar-wrapper">
+            <ul class="sidebar-nav">
+                <li class="sidebar-brand">
+                    <!-- <a href="#">
+                       List a New Home
+                    </a> -->
+                </li>
+                <li>
+                    <a href="#"><i class="fa fa-circle fa-5x" aria-hidden="true"></i><!-- <img src="/assets/chris.jpg" alt="Description"  class="side-nav-button"> --></a>
```

Comment:
Use the rails link helpers [link_to](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to)